ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -97,6 +97,19 @@ extension BrowserViewController: WKNavigationDelegate {
         hideReaderModeBar(animated: false)
       }
     }
+
+    if let tab = tab(for: webView) {
+      tab.redirectURLs = []
+    }
+    appendToRedirectChain(webView)
```

Comment:
nit: `// Add source URL to redirect chain`

Reply:
Fixed